jfrog-cli/utils/config/config.go

```diff
@@ -17,13 +18,14 @@ import (
 	"path"
 	"path/filepath"
 	"strings"
-	"fmt"
 )
 
 // This is the default server id. It is used when adding a server config without providing a server ID
 const (
-	DefaultServerId   = "Default-Server"
-	JfrogHomeEnv      = "JFROG_CLI_HOME"
+	DefaultServerId = "Default-Server"
+	JfrogHomeDirEnv = "JFROG_CLI_HOME_DIR"
+	JfrogHomeEnv    = "JFROG_CLI_HOME" // Old environment variable.
+	// Only used to support users relying on it before introducing JfrogHomeDirEnv
```

Comment:
It is best to put the description comment above the property it described. As it is now, it is a bit confusing, and may look like it is actually describing the *JfrogConfigFile* property.
In addition, in this specific case, it may be enough to have only
// Deprecated
as the description.